Kang-Minseokk/Invaders-SDP
# From Develop To Master : Final modify & sound added.

## What
This PR adds the login functionality for users. It includes the following changes:
- Added achievement sound effect.
- Resolve Null point Exception while repeating BGM sound file.

## Why
Issue created in #84 - Null point Exception

## Related Issue
#84

## How to Test
1. Start the application locally.
2. Sound check after clear a round while do not die.


## Screenshots

## Additional Information
...
